edtf/parser/parser_classes.py

```diff
@@ -27,7 +28,7 @@
 PRECISION_DAY = "day"
 
 
-def days_in_month(year, month):
+def days_in_month(year: int, month: int) -> dict:
```

Comment:
I'm getting a linting error that this returns an int not a dict.

Reply:
Fixed.